maniphttp/manipulator.go

```diff
@@ -667,15 +667,19 @@ func (s *httpManipulator) send(
 				goto RETRY
 
 			case io.ErrUnexpectedEOF, io.EOF:
-				lastError = manipulate.NewErrCannotCommunicate(snip.Snip(err, s.currentPassword()).Error())
+				if lastError == nil {
+					lastError = manipulate.NewErrCannotCommunicate(snip.Snip(err, s.currentPassword()).Error())
+				}
```

Comment:
👀 @primalmotion

Reply:
actually can you remove this, we have another more urgent manipulate PR that should go in master and 3.13. @CyrilPeponnet can you steal this code and add it to your PR?

Reply:
Done ✅ - https://github.com/aporeto-inc/manipulate/pull/115